Quick/Reporting/XCTestObservationCenter+Reporting.m

```diff
@@ -0,0 +1,137 @@
+//
+//  XCTestObservationCenter+Reporting.m
+//  Quick
+//
+//  Created by Paul Young on 1/2/15.
+//  Copyright (c) 2015 Brian Ivan Gesiak. All rights reserved.
+//
+
+#import "XCTestObservationCenter+Reporting.h"
+#import <XCTest/XCTestRun.h>
+#import "XCTestDriver.h"
+#import <objc/runtime.h>
+
+@implementation XCTestObservationCenter (Reporting)
+
++ (void)load {
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{
+        [self swizzleInstanceMethod:@selector(_testSuiteDidStart:) withMethod:@selector(qck_testSuiteDidStart:)];
```

Comment:
Here's where I imagine we'll have a conditional to check if a custom reporter is configured. If not, do nothing and let XCTest do it's thing.